### ISSUE TITLE
(fixed) Upgrading item (Golden Sword) not working

### ISSUE THREAD

**Issue by iorWout** [Jarne-Verlinden/test#3]
It looks like **upgrading** my **golden** **sword** is not working. I already tried to join another server, still the **same** result.

**Comment by Jarne-Verlinden** [comment]
Thank you for sending a bug report! I will have a look into it.

**Comment by Jarne-Verlinden** [comment]
Yes, you are right. It is a bug. Please give us some time to fix it. If it's still not fixed after a while, then please create a new one and refer to this issue. I will close it for now.

**Comment by Jarne-Verlinden** [comment]
It looks like many users are affected by this. Please wait while we fix this issue. This will be put on priority in our bugs list!

**Comment by Jarne-Verlinden** [comment]
Bug is fixed! Thank you all for your patience. All game servers will shutdown in a minute or two. I will also unpin this issue after some time. Thanks to @iorWout for reporting it!

**Comment by Jarne-Verlinden** [comment]
This was a test lol.
